flexget/plugins/search/limetorrents.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import unicode_literals, division, absolute_import
+from future.moves.urllib.parse import quote
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+
+import logging
+import re
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.requests import RequestException
+from flexget.utils.soup import get_soup
+from flexget.utils.search import torrent_availability
+from flexget.plugins.plugin_urlrewriting import UrlRewritingError
+
+log = logging.getLogger('limetorrents')
+
+class limetorrents(object):
+    """
+        limetorrents search plugin.
+    """
+
+    schema = {
+        'oneOf': [
+            {'type': 'boolean'},
+            {
+                'type': 'object',
+                'properties': {
+                    'category': {'type': 'string', 'enum': ['all', 'anime', 'applications', 'games', 'movies', 'music', 'tv', 'other'], 'default': 'all'},
+                    'order_by': {'type': 'string', 'enum': ['date', 'seeds'], 'default': 'date'}
+                },
+                'additionalProperties': False
+            }
+        ]
+    }
+
+    base_url = 'https://www.limetorrents.cc/'
+    errors = False
+
+
+    @plugin.internet(log)
+    def search(self, task, entry, config):
+        """
+            Search for entries on limetorents
+        """
+
+        if not isinstance(config, dict):
+            config = {'category': config}
+
+        order_by = ''
+        if isinstance(config.get('order_by'), str):
```

Comment:
I had a problem in the 1337x plugin where I was calling the config for order_by but since I allow you to not specify order_by it was bugging out - that was the fix I used in the last plugin, let me know if there is a better way